device/bgp/bgpsrtepolicycolorsubtlv.yaml

```diff
@@ -0,0 +1,15 @@
+components:
+  schemas:
+    BgpSrte.policyPrioritySubTlv:
+      description: >-
+        Configuration for the Policy Priority sub-TLV.
+        The Policy Priority to indicate the order in which the SR policies 
```

Comment:
The Policy Priority is used to indicate ...

Reply:
done